src/components/cta.tsx

```diff
@@ -11,8 +11,8 @@ export const CallToAction: FC = () => (
 				rel="noreferrer"
 			>
 				<span className="hidden sm:block">GitHub</span>
-				<span className="sm:hidden">
-					<ExternalLinkIcon size={16} />
+				<span className="sm:ml-2">
```

Comment:
```suggestion
				<span className="sm:ml-1">
```
`ml-2`

![image](https://github.com/user-attachments/assets/338f430b-4f27-4283-bf92-8b502bd21a5d)

`ml-1`

![image](https://github.com/user-attachments/assets/2538c9be-2d16-414d-b9ba-d19614d1b4fe)



`ml-1` is enough I guess.

Reply:
shadcn UI had ml-2/mr-2 for buttons with icons and it felt right thats why maintained it